### DISCUSSION TITLE
Is it a bug?

### DISCUSSION THREAD

**Discussion by MustacheMan2** [DarkflameUniverse/DarkflameServer#1119] [General]
Introducing: **Is it a bug** (more specifically, is it server-side, non-live accurate and fixable) or not?
First up, is it live accurate or not that the three Stromling Pirates by the pirate-only quickbuild in Gnarled Forest just fade in when spawning, and quite close to the player, rather than having their normal unearthing animation? I haven't found any live game footage of what their spawning is like yet.

I'll be continuing to post to this thread for similar questions as to not clutter the place, feel free to do the same.

**Comment by EmosewaMC** [comment]
https://youtu.be/CeFCzem2zrI?t=62
They did indeed just fade in.

**Comment by jejukin** [comment]
Yeah they're like a pirate ambush basically.  But you can quick build the siren before triggering their spawn, trivializing the encounter.

**Comment by MustacheMan2** [comment]
There is a noticeable change from live with how characters load in. Oftentimes characters visually load in much later than they should; this is most noticeable in rocket cutscenes and cannon cove shooting gallery. I got the feeling that it's just how the server operates and its not fixable, so checking in here before I make any bug reports.

**Reply by EmosewaMC** [reply]
Can you get some footage from live as a reference?

**Reply by MustacheMan2** [reply]
well, I feel a little dumb. Everything seems to be sound, but I swear it was delayed a while ago, so maybe an update improved it.

**Reply by EmosewaMC** [reply]
its possible it was improved but if it was that was not something we were trying to fix.  If you find footage create an issue and cc the footage.

**Comment by MustacheMan2** [comment]
I have no live game evidence of this, but I am quite certain the floating orbs in the ninjago lighting garden are supposed to harm the player in some way, currently there's no challenge to reaching the top. Maybe the `L_LIGHTNING_ORB_SERVER` script is related?

**Reply by EmosewaMC** [reply]
The script did knockback, which DLU already has an issue for not having implemented.

**Comment by MustacheMan2** [comment]
I am under the impression there are multiple team messages that are not implemented, such as pop ups for declining to join, and attempting to join a full team, and etc. but it's hard to find live footage of these things in action, otherwise I'd make a report. Also, the feature of joining a friend's instance via the map, but I don't know how it worked; can it teleport players to different zones entirely, or only to a different instance of the same world? Because if it's the latter there's a near zero chance that would be useful and not even worth looking into, but otherwise it could be useful.

**Reply by EmosewaMC** [reply]
> I am under the impression there are multiple team messages that are not implemented

They are implemented, but not all cases of the enum are implemented.  A fix implementing the missing replies is not hard, but time consuming and as such we have not prioritized it.

> pop ups for declining to join, and attempting to join a full team, and etc

Yes these should exist, no, no one has cared or wanted to put the time into implementing said feature since its QOL only and very low on the list of things people actually want.

> Also, the feature of joining a friend's instance via the map, but I don't know how it worked; can it teleport players to different zones entirely, or only to a different instance of the same world?

Only in the same zone. It serves no purpose on dlu since most servers do not see multiple instances of a zone at one time anyways, so this is not on the list of features to implement.

**Comment by MustacheMan2** [comment]
Is the Racemaster shirt not working like it should (5% speed increase), or is it just hard to tell because the speed increase is small? This is driving me nuts.

**Reply by EmosewaMC** [reply]
https://explorer.lu/skills/behaviors/31897/31898

It is setting your speed to be 510, which is overshadowed by the level 20 speedboost.  If you are below level 20 you should see a difference.  Same applies to the shock fruit in ninjago; sets speed to a value, not increase speed by x%.

**Comment by MustacheMan2** [comment]
I do believe the admiral's attack range is too large. It goes far beyond the visible animation. But I can't be sure. I recall live footage showing a mixed bag.

**Comment by MustacheMan2** [comment]
![Screenshot 2025-04-13 140631](https://github.com/user-attachments/assets/553ada39-ef4f-4a8b-8f50-a52c4078926a)
Go Outside and Play has the implication that progress between tiers should carry, but the LU explorer page potentially says otherwise (https://explorer.lu/missions/676) so I am unsure if the behavior seen in the screenshot is live accurate.

**Reply by EmosewaMC** [reply]
unlike every other complete x missions, this one is complete x missions vs complete x more missions.  Yes its a bug not sure if we have an issue yet or not.
